services/datamanager/builtin/builtin.go

```diff
@@ -79,13 +80,41 @@ type Config struct {
 	Tags                   []string                         `json:"tags"`
 	ResourceConfigs        []*datamanager.DataCaptureConfig `json:"resource_configs"`
 	FileLastModifiedMillis int                              `json:"file_last_modified_millis"`
+	SelectiveSyncer        string                           `json:"selective_syncer"`
```

Comment:
Perhaps `SelectiveSyncerName` to make it clear? Could you also test to see if it's possible to instantiate two sensors with the same name, and what would happen here?

Reply:
Talked offline and discussed that using the `FromDependencies` method to use the resource by name is fine for now, since this paradigm is repeated elsewhere and the UI explicitly does not allow configuring a two sensors with the same name.